mautrix_signal/portal.py

```diff
@@ -299,6 +299,7 @@ async def handle_matrix_message(
         self, sender: u.User, message: MessageEventContent, event_id: EventID
     ) -> None:
         if self.bridge.bridge_blocked:
+            await self.bridge._notify_bridge_blocked()
```

Comment:
I'd log to the terminal/log files first and then send the messages, as it sounds like an action that could throw or take a while.
Same with the other change in this file.

Reply:
Good call, done in 353da7e4498694b4818417ec31737470c16b0665